pkg/config/testdata/tracee-config.yml

```diff
@@ -0,0 +1,57 @@
+kind: TraceeConfig
+apiVersion: tracee.aquasecurity.github.io/v1alpha1
+signatures:
+  - metadata:
+      id: "CEL-2"
+      name: "Anti-Debugging"
+    eventSelectors:
+      - source: tracee
+        name: ptrace
+    expression: |
+      input.EventName == 'ptrace' and argument(input, 'request') == 'PTRACE_TRACEME'
+  - metadata:
+      id: "CEL-12"
+      name: "Illegitimate Shell"
+    eventSelectors:
+      - source: tracee
+        name: security_bprm_check
+        origin: "*"
+    expression: |
+      input.EventName == 'security_bprm_check' and
+      (
+        (arg(input, 'pathname') endsWith '/ash') or
```

Comment:
Can we add support for multiple values comparison? Will make our lives easier.

Reply:
Certainly, we'll look into existing signatures and add macros that make expression readable.

Reply:
Actually, CEL already defines a handful of macros to support iteration. We can write similar expression using the `exists` macro as follows:

```
['/ash', '/bash', '/csh', '/ksh', '/sh', '/tcsh', '/zsh', '/dash'].exists(e, input.stringArg('pathname').endsWith(e))
```